common/setups/rasr/gmm_system.py

```diff
@@ -1237,6 +1237,23 @@ def run_vtln_sat_step(self, step_args):
                     **step_args.sdm_args,
                 )
 
+    def run_forced_align_step(self, step_args):
+        train_corpus_keys = step_args.pop("train_corpus_keys", self.train_corpora)
```

Comment:
This will break old setups right? Since it is renamed from `corpus_keys` to `train_corpus_keys`. This is why it was not included in the other PR like this.

Reply:
what do you mean break? the hash should not be influenced or is it?

Reply:
further down you call `self.run_forced_align_step(step_args)` and just pass the args. But in the old code the args should contain: `step_args.pop("corpus_keys", self.train_corpora)`, but in the new code that key is called `"train_corpus_keys"`

Reply:
unless I am missing something I think old code using that function would now not work anymore.